pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -271,6 +272,57 @@ func (self *RefsHelper) CreateGitResetMenu(ref string) error {
 	})
 }
 
+func (self *RefsHelper) CreateCheckoutMenu(commit *models.Commit) error {
+	branches := lo.Filter(self.c.Model().Branches, func(branch *models.Branch, _ int) bool {
+		return commit.Hash == branch.CommitHash && branch.Name != self.c.Model().CheckedOutBranch
+	})
+
+	ref := commit.Hash
+	var menuItems []*types.MenuItem
+
+	menuItems = append(menuItems, &types.MenuItem{
+		LabelColumns: []string{fmt.Sprintf("%s %s", self.c.Tr.CheckoutCommit, self.c.Tr.AsDetachedHead), style.FgRed.Sprint(ref[:8])},
+		OnPress: func() error {
+			self.c.Confirm(types.ConfirmOpts{
```

Comment:
We don't need the confirmation any more; the menu already acts as a (sort of) confirmation now.